lib/dndfonts.sty

```diff
@@ -46,7 +60,7 @@
     section-family .initial:n        = \normalfont,
     section-family .value_required:n = true,
     section-style .tl_set:N         = \l__dnd_section_style_tl,
-    section-style .initial:n        = \linespread{.9} \color{titlered} \huge \scshape \RaggedRight,
+    section-style .initial:n        = \linespread{.9} \color{titlered} \large \scshape \RaggedRight,
```

Comment:
Why did you change the text size here? it is now the same size as `\subsubsection`.

The next size down from `\huge` is `\LARGE`. That would give a section header that was the same size as the basic rules' section header, but all of our fonts run .01in-.02in taller than the basic rules. I think it makes sense to keep that consistency across all the fonts.

Reply:
I did?! That must have been an accident.  Personally, I change it to `\LARGE`, but I'm not sure how this got into my Github repository.